tracing/jersey-client/src/main/java/module-info.java

```diff
@@ -40,7 +40,8 @@
     exports io.helidon.tracing.jersey.client;
 
     // needed to propagate tracing context from server to client
-    exports io.helidon.tracing.jersey.client.internal to io.helidon.tracing.jersey,io.helidon.microprofile.tracing;
+    exports io.helidon.tracing.jersey.client.internal to io.helidon.tracing.jersey,io.helidon.microprofile.tracing,
```

Comment:
Please revert, no need to export to trace exemplar.

Reply:
Fixed